diff --git a/packages/font-glyphs/src/letter/latin/upper-q.ptl b/packages/font-glyphs/src/letter/latin/upper-q.ptl
index 114ea5a..4abae48 100644
--- a/packages/font-glyphs/src/letter/latin/upper-q.ptl
+++ b/packages/font-glyphs/src/letter/latin/upper-q.ptl
@@ -82,10 +82,12 @@ glyph-block Letter-Latin-Upper-Q : begin
 	# Tails
 	define [QStraightTail df] : begin
 		local shift : StrokeWidthBlend 0 0.25
+		local rootX : df.middle - [HSwToV Stroke] * shift
+		local endX : [mix df.middle df.rightSB (2 / 3)] - [HSwToV Stroke] * shift
 		return : dispiro
 			widths.rhs
-			flat (df.middle - [HSwToV Stroke] * shift - TailDepth * 0.25) TailDepth [heading Upward]
-			curl (df.middle - [HSwToV Stroke] * shift) 0
+			flat endX TailDepth [widths.rhs.heading (Stroke * 1.08) Upward]
+			curl rootX 0 [widths.rhs Stroke]
 
 	define [QCurlyTail df] : begin
 		local shift : StrokeWidthBlend 0.5 0.6
